core/cold-wallet/wallet/generate-new-addresses.ts

```diff
@@ -0,0 +1,85 @@
+import { exit } from 'process';
+import { WalletHelper } from './wallet-helper';
+
+enum ExportType {
+  SQL = 'sql',
+  JSON = 'json',
+}
+
+const buildJSON = (index: number, address: string, wallet: string, accountIndex: number): string => {
+  return `{ "id": ${index}, "owner": "${address}", "ownerWallet": "${wallet}", "accountIndex": ${accountIndex} }\n`;
+};
+
+const buildSQL = (index: number, address: string, wallet: string, accountIndex: number): string => {
+  return `UPDATE table_name SET owner = '${address}', ownerWallet = '${wallet}', accountIndex = ${accountIndex} WHERE id = ${index}\n`;
+};
+
+const generate = async (
+  walletName: string,
+  numberOfAddresses: number,
+  startingIndex: number,
+  accountIndex: number,
+  exportType: ExportType,
+) => {
+  const wallet = await WalletHelper.restore();
+  wallet.initialize();
+
+  let result = '';
+
+  for (let i = 0; i < numberOfAddresses; i++) {
+    // in our new setup it doesn't matter if default address is used or not
+    // as it isn't used as collector address anymore
+    const address = await wallet.getAddress(i);
```

Comment:
Should it be `wallet.getAddress(startingIndex + i);``

Reply:
should be accountIndex + i. will fix it in another PR, because there are so many PRs depending on this one, that I would need to rebase all of them